build.gradle

```diff
@@ -63,8 +63,8 @@ android {
 
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_5
```

Comment:
Are you sure about this? Material Design is just a set of design guidelines, not limited to Android, nor even Java, and anyway, Java 7 is almost fully compatible with all Android versions: http://stackoverflow.com/questions/20480090/does-android-support-jdk-6-or-7